fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalCatalog.java

```diff
@@ -331,6 +336,13 @@ public boolean isEnableHmsEventsIncrementalSync() {
         return enableHmsEventsIncrementalSync;
     }
 
+    public Catalog getIcebergHiveCatalog() {
+        if (icebergHiveCatalog == null) {
```

Comment:
This method may be called concurrently

Reply:
for prevent this, i will move icebergHiveCatalog init to initLocalObjectsImpl()